src/test/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/integration/ResetLicenceTest.kt

```diff
@@ -53,9 +53,9 @@ class ResetLicenceTest : SqsIntegrationTestBase() {
     assertThat(events).hasSize(3)
 
     assertThat(events).extracting("user", "action", "details").containsExactly(
-      tuple("SYSTEM:API", "RESET", mapOf("bookingId" to 10)),
```

Comment:
`:` is not a legal character in a Java identifier so have changed this and other event user types to use `_` as a word separator instead. Hope changing the user type from `SYSTEM:API` (a type I think has been saved to the db already) to `SYSTEM_API` doesn't have negative consequences in the db? Seemed wrong to not make the API type an enum also.

Reply:
you can have a text value associated with the enum value.

Reply:
https://kotlinlang.org/docs/enum-classes.html

```kt
enum class Color(val colourName: String) {
    RED("Red"),
    GREEN("Green"),
    BLUE("Blue")
}
```
so you can then refer to the string label as:
`Color.RED.colourName`